tests/hps/test_hps_classification.py

```diff
@@ -191,6 +230,9 @@ def test_hparams_best_params_files_compatibility(
         "classifier_dropout",
     }
     for k in common_keys:
+        if k == "dataset_name_or_path":
+            continue
+
         if k in metadata:
             if isinstance(metadata[k], Enum):  # type: ignore
                 enum_hparam = getattr(type(metadata[k]), hparams[k])  # type: ignore
```

Comment:
at some that would be nice if these asserts would produce human readable info what to do if they don't pass

Reply:
It's in tests, therefore I don't think we need additional info about that check. Test name in my opinion is sufficent and the users should now where this error occurs.